services/appInsight_ui.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from "react";
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import {
+  ReactPlugin,
+  withAITracking,
+  AppInsightsContext,
+} from "@microsoft/applicationinsights-react-js";
+import { createBrowserHistory } from "history";
+
+let browserHistory = null;
+if (typeof document !== "undefined") {
+  browserHistory = createBrowserHistory();
+}
+
+const isDev = process.env.NODE_ENV === "development";
```

Comment:
I think we should load up AppInsights when a connection string is provided.

If one is not provided, log a message to console